ifrs17-template/Report/ActualsUseCaseReports.ipynb

```diff
@@ -0,0 +1,851 @@
+{
```

Comment:
This currently just shows the numbers. I will add text in a 2nd stage, as I find time.

Reply:
I would add it to the solution... It enhances it with more examples and information.
The question to me is just where.

We can have a dedicated folder for each Use Case, and then have all files there.
OR
We can use the folders we have, and spread the case into the many places.

Reply:
Went for the dedicated folder, as discussed.